setup.py

```diff
@@ -36,7 +36,7 @@
     zip_safe=False,
     install_requires=[
         'django>=3.0,<3.2',
-        'swapper @ git+git://github.com/openwisp/django-swappable-models.git@master',
+        'swapper~=1.3.0',
```

Comment:
please remove this because as I just said in the dev chat it's not needed anymore since it's defined in openwisp-utils.